{{cookiecutter.github_repository}}/provisioner/roles/postgresql/tasks/main.yml

```diff
@@ -29,11 +34,6 @@
     template: 'template0'
```

Comment:
@akarambir thanks for pointing it out, I missed that. But is there a specific reason to create database from `template0` as `template1` is the default one.